wordpress-mobile/WordPress-iOS
# Update WordPressAuthenticator Pod to 1.1.7

This PR updates the WordPressAuthenticator pod to the latest release. It makes some properties configurable for WooCommerce, and makes no visual changes to WordPress.

**To test:**
1. Check out this branch
2. Clean the project
3. `pod install` / `pod update`

Before / After: the instruction strings found on email and site URL login should look exactly the same as before:
<img src="https://user-images.githubusercontent.com/1062444/49592052-bf211780-f935-11e8-9006-4aaeca19a276.png" width="350" />
<img src="https://user-images.githubusercontent.com/1062444/49592055-c2b49e80-f935-11e8-89ef-e60654f7fb60.png" width="350" />
